resources/hu/cemu.po

```diff
@@ -1994,8 +1994,8 @@ msgid "&Render upside-down"
 msgstr "&Fejjel lefelé renderelés"
 
 #: MainWindow.cpp
-msgid "&Optimize shader SPIR-V"
-msgstr "&Shader SPIR-V optimalizálása"
+msgid "&Async compile (Vulkan)"
+msgstr "&Szinkronizáltalan fordítás (Vulkan)"
```

Comment:
Looking at [Microsoft Terminology database](https://www.microsoft.com/en-us/language/Search?&searchTerm=asynchronous&langID=382&Source=true&productid=undefined), I've seen that in their products _asynchronous_ is translated into Hungarian as _aszinkron_.
Also, Google Translate suggests to use _szinkronizálatlan_ instead of _szinkronizáltalan_. In fact, searching on Google for the latter yields only a single result (typo?).